apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/tour-card.js

```diff
@@ -93,12 +149,59 @@ function CardOverlayControls( { onMinimize, onDismiss } ) {
 					className="welcome-tour-card__minimize-icon"
 					icon={ minimize }
 					iconSize={ 24 }
-					onClick={ () => onMinimize( true ) }
+					onClick={ handleOnMinimize }
 				></Button>
 				<Button isPrimary icon={ close } iconSize={ 24 } onClick={ () => onDismiss() }></Button>
 			</Flex>
 		</div>
 	);
 }
 
+function TourRating() {
+	let isDisabled = false;
+	const tourRating = useSelect( ( select ) => select( 'automattic/nux' ).tourRating() );
+	const { setTourRating } = useDispatch( 'automattic/nux' );
+
+	if ( ! isDisabled && tourRating ) {
+		isDisabled = true;
+	}
+	const rateTour = ( isThumbsUp ) => {
+		if ( isDisabled ) {
+			return;
+		}
+		isDisabled = true;
+		setTourRating( 'thumbs-up' );
+		recordTracksEvent( 'calypso_editor_wpcom_tour_rate', {
+			thumbs_up: isThumbsUp,
+			is_gutenboarding: window.calypsoifyGutenberg?.isGutenboarding,
+		} );
+	};
+
+	return (
+		<>
+			<p className="welcome-tour__end-text">Did you find this guide helpful?</p>
+			<div>
+				<Button
```

Comment:
We should add an `aria-label` to these buttons. The icons are set as `aria-hidden` by Gutenberg, so currently these currently appear as label-less buttons in a screen reader.